framework/src/modules/chain/dpos/account.js

```diff
@@ -0,0 +1,263 @@
+const BigNum = require('@liskhq/bignum');
+
+const _mergeRewardsAndDelegates = (delegatePks, rewards) =>
+	delegatePks.reduce((arr, delegatePk, index) => {
+		let delegate = arr.find(d => d.publicKey === delegatePk);
```

Comment:
it's obvious from interpreting the whole line but I would still name the local variables `delegate` inside the anonymous function.